test/dubhash.test.ts

```diff
@@ -0,0 +1,108 @@
+import { promises } from 'fs';
+import * as path from 'path';
+import { SignatureResult } from '../lib/types';
+import { getDubHashSignature } from '../lib/dubhash';
+const { readFile } = promises;
+
+describe('getDubHashSignature', () => {
+  it('returns correct SignatureResult for binary file', async () => {
+    const fixturePath = path.join(__dirname, 'fixtures');
+    const filePath = path.join(fixturePath, 'dubhash', 'test02');
+    const fileContents = await readFile(filePath);
+    const expected: SignatureResult = {
+      path: filePath,
+      hashes_ffm: [
+        {
+          format: 1,
+          data: 'aT6a+E09/MceZA4AW9xeLg', //TODO: Verify with Daniel
```

Comment:
regarding the TODO: i think we verified this one (you reported the issue and we fixed it)
iirc

Reply:
You are correct. Removing the comment now.